src/views/ReferralTerms/ReferralTerms.js

```diff
@@ -14,23 +14,23 @@ export default function ReferralTerms(props) {
       <div className="default-container Page page-layout">
         <div>
           <div className="Page-title-section center">
-            <div className="Page-title">GMX Referral Program</div>
+            <div className="Page-title">MYC Referral Program</div>
             <div className="Page-subtitle">Terms and Conditions</div>
             <div className="Page-description">Last modified: May 5th, 2022</div>
           </div>
           <div className="content">
             <div className="section">
               <p className="body-text">
-                Our greetings and welcome to GMX.io (“we” “us,” or “our”), the informational resource for GMX Protocol.
+                Our greetings and welcome to MYC.io (“we” “us,” or “our”), the informational resource for MYC Protocol.
```

Comment:
Legal may have to look over the terms, but maybe myc.io has to become mycelium.xyz otherwise the terms may not be applicable?

Reply:
I think these are old terms regardless. They arent displayed anywhere, might revert back to displaying GMX in here. Keeping because I think webby wants to revive referrals eventually